server/base/src/main/java/org/apache/accumulo/server/conf/codec/VersionedProperties.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.server.conf.codec;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * Version properties maintain a {@code Map<String,String>}; of property k,v pairs along with
+ * versioning information metadata.
+ * <p>
+ * The metadata used to verify cached values match stored values. Storing the metadata with the
+ * properties allows for comparison of properties and can be used to ensure that vales being written
+ * to the backend store have not changed. This metadata should be written / appear early in the
+ * encoded bytes and be uncompressed so that decisions can be made that may make deserialization
```

Comment:
```suggestion
 * properties allows for comparison of properties and can be used to ensure that values being
 * written to the backend store have not changed. This metadata should be written / appear early in
 * the encoded bytes and be uncompressed so that decisions can be made that may make deserialization
```
The missing 'u' in values pushed these to new lines